homepage.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Homepage</title>
+  <link rel="stylesheet" href="style.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link
+    href="https://www.dafontfree.net/embed/Y29jb2dvb3NlLXByby10aGluLXJlZ3VsYXImZGF0YS84NS9jLzE3NjcxMS9Db2NvZ29vc2UgUHJvIFRoaW4tdHJpYWwudHRm"
+    rel="stylesheet" type="text/css" />
+  <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+</head>
+
+<body>
+
+  <header class="header">
+    
+    <nav>
+
+      <div class="topNav">
+
+        <ul class="navSocials">
+          <li><a href="#"><img src="https://img.icons8.com/color/48/000000/facebook-new.png" alt="facebook" /></a>
+          </li>
+          <li><a href="#"><img src="https://img.icons8.com/fluency/48/000000/instagram-new.png" alt="instagram" /></a>
+          </li>
+          <li><a href="#"><img src="https://img.icons8.com/color/48/000000/twitter--v1.png" alt="twitter" /></a></li>
+          <li><a href="#"><img src="https://img.icons8.com/glyph-neue/64/000000/github.png" alt="github" /></a></li>
+          <li><a href="#"><img src="https://img.icons8.com/color/48/000000/linkedin.png" alt="linkedin" /></a></li>
+        </ul>
```

Comment:
- [ ] Please follow the instructions and fix what the previous review said: "The top short bar section with social media links, logout, and language links aligned to the right"
- Original Design
![b4](https://user-images.githubusercontent.com/63972670/163487582-a92db681-0ac9-4846-8dfd-7a6eab34e288.PNG)
- Your Design


![b5](https://user-images.githubusercontent.com/63972670/163487768-e4b285b3-6291-44ad-a468-0048f0bd64af.PNG)